File: examples/eApi v1.0/dotNET/KHGTWDemo/KHGatewayClientExample/Communication/DataObjects/EchoRequest.cs
using System.Text;

namespace KHGatewayClientExample.Communication.DataObjects;

public class EchoRequest : SignBaseRequest
{
    public override string ToSign()
    {
        var sb = new StringBuilder();
        Add(sb, MerchantId);
        Add(sb, Dttm);
        DeleteLast(sb);
        return sb.ToString();
    }
}

File: examples/eApi v1.0/java/eAPI-v10-hu-connector/src/main/java/cz/monetplus/mips/eapi/v10hu/connector/entity/EchoRequest.java
package cz.monetplus.mips.eapi.v10hu.connector.entity;


public class EchoRequest extends SignBase {

	private static final long serialVersionUID = -3825192932302805075L;
	
	public String merchantId;
	
	public EchoRequest() {
	}
	
	public EchoRequest(String merchantId) {
		this.merchantId = merchantId;
	}

	@Override
	public String toSign() {
		StringBuilder sb = new StringBuilder(); 
		add(sb, merchantId);
		add(sb, dttm);
		deleteLast(sb);
		return sb.toString();
	}

}
